src/test/ui/lint/packed_reference.rs

```diff
@@ -0,0 +1,20 @@
+#![deny(packed_references)]
+
+#[repr(packed)]
+pub struct Good {
+    data: &'static u32,
+    data2: [&'static u32; 2],
+    aligned: [u8; 32],
+}
+
+fn main() {
+    unsafe {
+        let good = Good { data: &0, data2: [&0, &0], aligned: [0; 32] };
+
+        let _ = &good.data; //~ ERROR reference to packed field
```

Comment:
Please also add a line for `let _: *const &'static u32 = &good.data;`

Reply:
Done.